### ISSUE TITLE
FreshRSS should (mainly) work disabled JavaScript

### ISSUE THREAD

**Issue by math-GH** [marienfressinaud/FreshRSS#4148]
As discussed here: https://github.com/FreshRSS/FreshRSS/pull/4125#issuecomment-1010325581

The main functions of FreshRSS should (mainly) work without JavaScript.

main functions = difficult to define, but would say something like: login, read and add/config feeds.

Let's discuss this goal here and collect the issues.

**Comment by math-GH** [comment]
One issue is (without JavaScript):

The user cannot login. Because of the crypt JavaScript script.

**Comment by Frenzie** [comment]
The login probably looks like the primary issue to me, since logging into view stuff does sound like fairly basic functionality. The crypt stuff is probably mainly intended for regular HTTP since everything should normally already be encrypted over HTTPS?

Adding a couple of Netsurf screenshot for reference. (The "load more articles" works very well.) It's something I like to keep the occasional eye on. Either this particular theme is worse than Origine or we broke it a bit more than it used to be broken though over the last year or so.
![Screenshot_2022-01-12_20-31-05](https://user-images.githubusercontent.com/202757/149208855-61c83a94-49aa-483c-9ccb-699d1237bdcf.png)
![Screenshot_2022-01-12_20-36-16](https://user-images.githubusercontent.com/202757/149209580-1fd0c703-6ad0-4fe3-99c1-58dfaa79568c.png)

(At least personally my thoughts go more towards Netsurf and Links2 than towards NoScript.)

Links2 doesn't listen to display:none, so there's no real difference between regular and reading view:
![Screenshot_2022-01-12_20-34-25](https://user-images.githubusercontent.com/202757/149209311-e8bed0c3-6541-463c-86d4-8e5f709a2086.png)

**Comment by Alkarex** [comment]
> One issue is (without JavaScript): The user cannot login. Because of the crypt JavaScript script.

Only the Web Form login method requires JavaScript. The HTTP Basic Auth and No Auth methods do not

**Comment by Frenzie** [comment]
Indeed, yesterday I just looked at it without JS but you simply have to make a minor adjustment in the configuration. :-)

**Comment by math-GH** [comment]
Why/For what is JS on the login screen (still) needed?

**Comment by Alkarex** [comment]
JavaScript on the Web form login method is mostly for cases without HTTPS.
It adds another layer of protection by not transmitting the password, only a response to a challenge
